identity/webapp/src/frontend/MyAccount/MyAccount.tsx

```diff
@@ -53,14 +66,38 @@ const AccountStatus: React.FC<React.ComponentProps<typeof StatusAlert>> = ({ typ
   );
 };
 
+async function fetchRequestedItems(userId): Promise<RequestsList | undefined> {
+  try {
+    const response = await fetch(`/api/users/${userId}/item-requests`);
+    const json = await response.json();
+    return json;
+  } catch (e) {
+    console.log(e);
+  }
+}
+
 const Profile: React.FC = () => {
   const history = useHistory();
   const { user, isLoading, update } = useUserInfo();
   const [isEmailUpdated, setIsEmailUpdated] = useState(false);
   const [isPasswordUpdated, setIsPasswordUpdated] = useState(false);
+  const [requests, setRequests] = useState<RequestsList | undefined>();
+  const allowedRequests = 10;
+
+  useEffect(() => {
+    async function fetchRequests() {
+      if (user && user.userId) {
+        const items = await fetchRequestedItems(user.userId);
+        setRequests(items);
+      }
+    }
+    fetchRequests();
+  }, [user]);
```

Comment:
I am quite confused about when these effects are going to get called. In this example, when does user change? Should we havbe this on `user.userId` as I am assuming it's when someone logs in, although I don't suppose that's async?

Maybe we should enforce this?
https://github.com/facebook/react/issues/14920